examples/falcon_host_details/README.md

```diff
@@ -0,0 +1,34 @@
+# Falcon Host Details
+
+Stand-alone tool that uses Host API to query all the host details and output JSON to the stdout. This tool can be used together with JSON parsing tools like `jq` in order to build reports of your liking.
```

Comment:
Can we provide something that optionally doesn't rely on jq?

Reply:
We can. But I would argue we shouldn't. We would need to introduce and maintain json parsing interface that would end-up being always inferior to `jq`.

Besides, when I was young people kept saying the following.
- Write programs that do one thing and do it well.
- Write programs to work together.
- Write programs to handle text streams, because that is a universal interface.

Reply:
I thought so, but just thought I would ask the question in case there was something that could easily be done.

> Besides, when I was young people kept saying the following.

But that was when you were young.... :D